### DISCUSSION TITLE
I can't get bios.bin

### DISCUSSION THREAD

**Discussion by yama497** [chipsec/chipsec#2336] [Q&A] [answered]
I bought mini-PC Ace Magic S1 (Intel N100) last month. I tried to get bios.bin. But I can't.  My friends says,"You shouldn't use the PC."
How should I do?  Sorry for my poor English.  Thanks in advance.

################################################################
################################################################
[CHIPSEC] Version  : 1.13.6
[CHIPSEC] Arguments: spi dump ace_magic.bin

ERROR: Unknown Platform: VID = 0x8086, DID = 0x461C, RID = 0x00, CPUID = 0xB06E0
ERROR: Platform is not supported (Unknown Platform: VID = 0x8086, DID = 0x461C, RID = 0x00, CPUID = 0xB06E0).
ERROR: To specify a cpu please use -p command-line option
ERROR: To specify a pch please use --pch command-line option

ERROR: If the correct configuration is not loaded, results should not be trusted.

**Comment by npmitche** [comment, marked as answer]
Hi, This is because Intel has not yet published the Datasheet for this platform. Once the datasheet has been published, we will work on an XML configuration for it. You can try to use `-p adl` as adln is a derivative of adl. Just note if it detects an issue, it might not be valid.

**Reply by yama497** [reply]
Thank you very much for your reply.  I'm sorry I am a complete amateur.  So I can't understand how to use "-p adl".
Would you tell me how I should describe a command?

**Reply by yama497** [reply]
"to use -p adl" seens far beyond my knowledge. I think I will wait until the Database for the platform is published and I get enogh knowledge about running chipsec .
I'm very grateful to you for your kindness.

**Reply by yama497** [reply]
I could use -p adl and get bios.bin!  I'm very grateful to you for your advice.
